horizon/enforcer/api.py

```diff
@@ -276,32 +276,46 @@ async def conditional_is_allowed(
     *,
     policy_package: str = MAIN_POLICY_PACKAGE,
     external_data_manager_path: str = "/check",
+    external_data_manager_method: str = "POST",
```

Comment:
Isn't it better to create an alternative function at this point specifically for the external data manager?

Reply:
I added an alternative function, this is the `_is_allowed_data_manager` but I didn't want to do the repeated if of enable external data manger so I created a function that accepts the relevant params and performs the call to the relevant system